pkg/liqo-controller-manager/namespace-controller/check_LiqoLabel_presence.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Would avoid capital letters in the file name, for consistency (as underscores are also used)

Reply:
sure, thanks